apps/hexagon_dma/process_rawplanar.cpp

```diff
@@ -0,0 +1,94 @@
+#include <stdio.h>
+#include <memory.h>
+#include <assert.h>
+#include <stdlib.h>
+#include "halide_benchmark.h"
+#include "pipeline_rawplanar.h"
+#include "HalideRuntimeHexagonDma.h"
+#include "HalideBuffer.h"
+#include "../../src/runtime/mini_hexagon_dma.h"
+
+int main(int argc, char **argv) {
+    if (argc < 3) {
+        printf("Usage: %s width height\n", argv[0]);
+        return 0;
+    }
+
+    const int width = atoi(argv[1]);
+    const int height = atoi(argv[2]);
+
+    // Fill the input buffer with random data. This is just a plain old memory buffer
+    const int buf_size = width * height * 4;
+    uint8_t *data_in = (uint8_t *)malloc(buf_size);
+    for (int i = 0; i < buf_size;  i++) {
+        data_in[i] = ((uint8_t)rand()) >> 1;
+    }
+
+    Halide::Runtime::Buffer<uint8_t> input(nullptr, width, height, 4);
+ 
+    input.raw_buffer()->dim[0].stride = 1;
```

Comment:
Don't need any of this + 5 lines (this is all the default).

Reply:
Removed extra lines